src/sentry/sentry_metrics/consumers/indexer/batch.py

```diff
@@ -71,6 +73,12 @@ def invalid_metric_tags(tags: Mapping[str, str]) -> Sequence[str]:
     return invalid_strs
 
 
+def extract_use_case_id(mri: str) -> Optional[str]:
+    if (matched := MRI_RE_PATTERN.match(mri)) is not None:
+        return matched.group(2)
```

Comment:
I mentioned this in my other comment, but in the 2 cases we have right now, performance and release health, the namespace (which is `matched.group(2)` here) is going to be "transactions" and "sessions" respectively. Let's make sure we check for these 2 cases and return "performance" and "release-health" instead.